src/JavaSession1/ArrayIntersection.java

```diff
@@ -0,0 +1,71 @@
+package JavaSession1;/*Question 5: Find common elements between two arrays.*/
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ArrayIntersection {
+
+    public static void main(String[] args) {
+
+        int[] firstArray;
+        int[] secondArray;
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("Enter Size of First Array");
+        int firstArraySize = sc.nextInt();
+        firstArray = new int[firstArraySize];
+        insertInArray(firstArray, firstArraySize);
+
+
+        System.out.println("Enter Size of Second Array");
+        int secondArraySize = sc.nextInt();
+        secondArray = new int[firstArraySize];
+        insertInArray(secondArray, secondArraySize);
+
+
+        ArrayList<Integer> intersectionElement = arrayIntersection(firstArray, secondArray);
+
+        System.out.println("First Array is : ");
+        printArray(firstArray);
+
+        System.out.println("Second Array is : ");
+        printArray(secondArray);
+
+        System.out.println("Intersected Element Are:");
+        intersectionElement.forEach(System.out::println);
+
+
+    }
+
+    private static ArrayList<Integer> arrayIntersection(int[] firstArray, int[] secondArray) {
+        ArrayList<Integer> elementList = new ArrayList<>();
+        for (int firstElement :
+                firstArray) {
+            for (int secondElement : secondArray) {
+                if (firstElement == secondElement)
+                    elementList.add(firstElement);
+            }
```

Comment:
Variable name firstElement and secondElement does not seems to be appropriate. These variable are changing it's valued during the iteration so you should keep the name defining its purpose or can keep the default ones at least.